lib/steps.js

```diff
@@ -418,149 +424,33 @@ When(/^(?:the user gets)?(?:I get)? to ?(?:the)?(?: ?question)?(?: id)? "([^"]+)
 
   // Add full used table to report so it can be copied if needed.
   // TODO: Should we/How do we get this in the final scenario report instead of here?
-  reports.addToReport( scope, { type: `note`, value: `\n${ ' '.repeat(2) }Rows that got set:` });
-  reports.addToReport( scope, { type: `step`, value: `${ ' '.repeat(4) }And I get to the question id "${ target_id }" with this data:` });
-  reports.addToReport(scope, { type: `row`, value: `${ ' '.repeat(6) }| var | value | trigger |`, });
+  reports.addToReport( scope, { type: `note info`, code: `ALK0063`, value: `\n${ ' '.repeat(2) }Rows that got set:` });
+  reports.addToReport( scope, { type: `step info`, code: `ALK0064`, value: `${ ' '.repeat(4) }And I get to the question id "${ target_id }" with this data:` });
+  reports.addToReport(scope, { type: `row info`, code: `ALK0065`, value: `${ ' '.repeat(6) }| var | value | trigger |`, });
   let at_least_one_row_was_NOT_used = false;
   for ( let row of sorted_rows ) {
     if ( row.times_used > 0 ) {
-      reports.addToReport(scope, { type: `row`, value: row.report_str, });
+      reports.addToReport(scope, { type: `row info`, code: `ALK0066`, value: row.report_str, });
     } else {
       at_least_one_row_was_NOT_used = true;
     }
   }
 
   // Add unused rows if needed
   if ( at_least_one_row_was_NOT_used ) {
-    reports.addToReport( scope, { type: `note`, value: `${ ' '.repeat(2) }Unused rows:` });
+    reports.addToReport( scope, { type: `note info`, code: `ALK0067`, value: `${ ' '.repeat(2) }Unused rows:` });
     for ( let row of sorted_rows ) {
       if ( row.times_used === 0 ) {
-        reports.addToReport(scope, { type: `row`, value: row.report_str, });
+        reports.addToReport(scope, { type: `row info`, code: `ALK0068`, value: row.report_str, });
       }
     }
   } else {
-    reports.addToReport(scope, { type: `note`, value: `${ ' '.repeat(2) }All rows were used` });
+    reports.addToReport(scope, { type: `note info`, code: `ALK0069`, value: `${ ' '.repeat(2) }All rows were used` });
   }
 
-  reports.addToReport(scope, { type: `note`, value: `` });
+  reports.addToReport(scope, { type: `note info`, code: `ALK0070`, value: `` });
```

Comment:
This is cosmetic and doesn't need a code (it's just adding a new line).

Reply:
Then again, we don't have another way to add a new line to the report and the report is going to add a code to every line in the debug...

Reply:
Fixed